content/sensu-go/6.1/sensuctl/back-up-recover.md

```diff
@@ -174,13 +174,17 @@ sensuctl describe-type all
   core/v2.Event                  events                core/v2             Event                true
   core/v2.EventFilter            filters               core/v2             EventFilter          true
   core/v2.Handler                handlers              core/v2             Handler              true
-  core/v2.Hook                   hooks                 core/v2             Hook                 true
+  core/v2.HookConfig             hooks                 core/v2             HookConfig           true
   core/v2.Mutator                mutators              core/v2             Mutator              true
   core/v2.Role                   roles                 core/v2             Role                 true
   core/v2.RoleBinding            rolebindings          core/v2             RoleBinding          true
   core/v2.Silenced               silenced              core/v2             Silenced             true  
 {{< /code >}}
 
+{{% notice note %}}
+**NOTE**: In Sensu 6.1.0, for hooks, the fully qualified name is `core/v2.Hook` and the type is `Hook`.
+{{% /notice %}}
```

Comment:
So in 6.1.0 and previous versions, it wouldn't even work at all with `core/v2.Hook`, so I'm wondering if we should modify this note somehow, so users don't expect it to work?